### PR TITLE
Cypress/E2E: Remove duplicate test executions and fix TM4J key

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Remove duplicate test executions and fix TM4J key

This PR fixes TM4J integration for: MM-T690 up to MM-T695, MM-T698, MM-T699 and MM-T703

For MM-T703, I've combined the test script from MM-T704 and then archive MM-T704.

### CODE REVIEW

File: e2e/cypress/integration/integrations/custom_slash_commands/custom_slash_commands_spec.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ import {
     saveConfigForScheme,
 } from '../../enterprise/system_console/channel_moderation/helpers';
 
-import {addNewCommand} from './slash_commands_spec';
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/custom_slash_commands/custom_slash_commands_spec.js:25]: This import causes the tests from slash_commands_spec to run in this spec file.

**Review by josephbaylon — APPROVED** [review]
LGTM. Tests passed locally. Thanks @saturninoabril !!

**Comment by saturninoabril** [comment]
@iomodo Gentle ping for your review if you get a chance. Just hoping to get in into release-5.32 before running automation test for RC1. Thanks!

**Review by iomodo — APPROVED** [review]
LGTM!

**Comment by iomodo** [comment]
@saturninoabril sorry for the late review, I was on PTO last week

**Comment by saturninoabril** [comment]
Oh, sorry for the ping. I shouldn't requested for your review in the first place. Thanks for your review @iomodo and hope you had a great week!

**Pull request merged**